exercises/javascript-1-a/README.md

```diff
@@ -0,0 +1,34 @@
+## Introduction
+
+This is Part 1 of our research into how people write JavaScript differently. This exercise should take 10-15 minutes. There is no right or wrong way to approach it - just do what feels the most natural to you. Feel free to use books, Google or Stack Overflow, just like you would if you were programming normally, but please don't actively look for a solution to the problem as this will negatively affect the research.
+
+## Instructions
+
+You're a magician and you handle a deck of cards. In order to correctly execute your magic trick, you need to be able to _move_ a card from one position to another position. That is, you need to be able to `rearrange` the deck. Naturally, you want to be able to move in _both directions_, as well as use a negative numbers for the `from` and `to` values (so you can say "from the top" or "from the bottom" -- it's all about the magic ✨).
+
+Create a function `arrange` that takes:
+
+- an `array` (of length `n`) of items,
+- a `from` position (whole `number`, `-n <= from < n`),
+- and `to` position (whole `number`, `-n <= to < n`)
+
+It returns a _new array_ with the `item` moved from the position `from` to the position `to`:
```

Comment:
```suggestion
Positive numbers means that you move cards from the bottom of the deck (the left hand side) and negative numbers refer to starting at the top of the deck (the right hand side).

It returns a _new array_ with the `item` moved from the position `from` to the position `to`:
```

Reply:
Also, I have no idea if the top of the deck should be the LHS or the RHS. I can see strong arguments for both ways, so I think we just need to define it.

Reply:
Right. That sorta makes sense. Let's do this!